pkg/daemon/ceph/client/erasure-code-profile.go

```diff
@@ -91,6 +91,17 @@ func CreateErasureCodeProfile(context *clusterd.Context, clusterName string, con
 	return nil
 }
 
+func DeleteErasureCodeProfile(context *clusterd.Context, clusterName string, erasureCodeProfile string) error {
+	args := []string{"osd", "erasure-code-profile", "rm", erasureCodeProfile}
+
+	_, err := ExecuteCephCommand(context, clusterName, args)
```

Comment:
is there any useful output that we are going to miss here?  often, the details of a failure don't come back in the error object.

Reply:
when i ran the command to delete it, i didnt get an output. But i do get an error message and non zero error code when the deletion fails. For example if you try to delete it while the pool still exist.

I was just following the same pattern as https://github.com/rook/rook/blob/master/pkg/daemon/ceph/client/erasure-code-profile.go#L86. I could read the output and do something like `if out != nil && out != ""` but im not sure of a good way to determine an output log vs an actual error.

Reply:
i think it'd be sufficient to just include the `out` return value in the `fmt.Errorf` call when `err != nil`.  no need to make decisions based on the value of `out`, just include it when there is an error that we are returning here.